rotkehlchen/db/upgrades/v32_v33.py

```diff
@@ -61,11 +61,21 @@ def _refactor_xpubs_and_xpub_mappings(cursor: 'Cursor') -> None:
     cursor.execute('ALTER TABLE xpub_mappings_copy RENAME TO xpub_mappings;')
 
 
+def _create_new_tables(cursor: 'Cursor') -> None:
+    cursor.execute("""
+        CREATE TABLE IF NOT EXISTS address_book (
+            address TEXT NOT NULL PRIMARY KEY,
+            name TEXT NOT NULL
```

Comment:
you forgot the blockchain in the DB upgrade part.

Reply:
Ouch. Fixed